scenes/menus/storybook/components/storybook.gd

```diff
@@ -37,8 +37,8 @@ func _enumerate_quests() -> Array[Quest]:
 
 	for dir in ResourceLoader.list_directory(quest_directory):
 		var quest_path := quest_directory.path_join(dir).path_join(QUEST_RESOURCE_NAME)
-		if ResourceLoader.exists(quest_path, "Quest"):
-			var quest: Quest = ResourceLoader.load(quest_path, "Quest")
+		if ResourceLoader.exists(quest_path):
+			var quest: Quest = ResourceLoader.load(quest_path)
```

Comment:
Hmm, interesting caveat of the ResourceLoader. Good explanation in the commit message.